src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +66,33 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        try {
+            authenticationService.register("emailExample@mail.com", "1234");
+        } catch (RegistrationException e) {
+            e.printStackTrace();
```

Comment:
bad practice)

Reply:
I do not agree with you here. This is just a test, in which I would simply see a exception message from the service when running.